contracts/multisig/src/msg.rs

```diff
@@ -21,6 +22,13 @@ pub enum ExecuteMsg {
     StartSigningSession {
         worker_set_id: String,
         msg: HexBinary,
+        chain_name: ChainName,
+        /* Address of a contract responsible for signature verification.
+        The multisig contract verifies each submitted signature by default.
+        But some chains need custom verification beyond this, so the verification can be optionally overridden.
+        If a callback address is provided, signature verification is handled by the contract at that address
+        instead of the multisig contract. TODO: define interface for callback */
+        sig_verification_callback_address: Option<String>,
```

Comment:
```suggestion
        sig_verifier: Option<String>,
```
This is what I would name it with.

Reply:
do we just use string for addresses in other contexts or should this be an address type?

Reply:
We just use string and then convert to an address type, since addresses need to be validated via `deps.api.addr_validate()` and so we can't use serde to validate the address